sdk/python/tests/integration/feature_repos/universal/data_sources/file.py

```diff
@@ -46,3 +49,79 @@ def create_offline_store_config(self) -> FeastConfigBaseModel:
 
     def teardown(self):
         self.f.close()
+
+
+class S3FileDataSourceCreator(DataSourceCreator):
+    f: Any
+    minio: DockerContainer
+    bucket = "feast-test"
+    access_key = "AKIAIOSFODNN7EXAMPLE"
```

Comment:
Can't really get around hardcoding `access_key` and `secret` in two places right now. The test suite doesn't allow us to inject configuration. cc @achals it would be nice if we could propagate a config (context-like) object.

Reply:
Agreed yeah I'll make a task for that